setup.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+from setuptools import find_packages, setup
+
+REQUIRES = [
+    'ddt',
+    'fauxfactory',
+    'nailgun==0.9.0',
```

Comment:
Can we require [0.10.0](https://github.com/SatelliteQE/nailgun/releases/tag/0.10.0) instead? #2096 and #2103 both make use of that release. Also, both version 0.9.0 and 0.10.0 are backward-compatible with 0.8.0, so there's little risk of breakage.